### ISSUE TITLE
Problem with "-" as separator

### ISSUE THREAD

**Issue by maitrid** [vot/xkpasswd-node#6]
It seems when "-" is specified as the separator, the result is not limited to "-". I see "-", "_", "+" and "#". (BTW, sorry the wrong subject and description was written here a moment ago)

**Comment by vot** [comment]
Hi maitrid,

Thanks for raising the issue. What is the syntax you're using?

I just checked what results I get for `xkpasswd -s=-` (using an equals sign specifically rather than a space) and it produces the desired effects on my machine.

*Update:* I added a note to readme about using the equals sign in that case. This is due to the way arguments are parsed - I'll look into how this can be dealt with (that parsing is done by my other module, clarg).

Let me know if this is a reasonable solution to your problem.
Cheers
